pulseapi/profiles/migrations/0023_auto_20190807_1733.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-08-07 17:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0022_userprofile_created_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userprofile',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True),
```

Comment:
immediate second migration that changes the identity of `created_at` to one that can only ever be "whatever the date/time is right now" when a profile gets created.